### DISCUSSION TITLE
Modular CLI 0.4.1 and Mojo 0.7.0 released!

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1667] [Announcements]
Hi everyone, Modular 💻 0.4.1 and Mojo 🔥 0.7.0 are live! 🚀

Go to https://modul.ar/download for instructions about how to upgrade or install.

It's a big release! Check out the full changelog here: https://modul.ar/changelog

And a blog about the new features: https://modul.ar/v0-7

**Comment by tristanbiesecker** [comment]
I think you'll also want to update the pinned discussion on the discussions page to reflect this release (it's still showing the announcement for 0.6.1).

**Reply by gryznar** [reply]
@jackos

**Comment by Eprahim-taha** [comment]
There is not enough documentation to explain the references  🥺

**Reply by gryznar** [reply]
Explaining them in current state may become quickly obsolete. As changelog states it is early version and they are not working currently as expected to work

**Comment by Eprahim-taha** [comment]
Why isn't Mojo working 🔥 0.7 on replit.com or Google colab
